src/content/perf/ui-performance.md

```diff
@@ -38,6 +38,12 @@ steps to take, and tools that can help.
 
 ## Diagnosing performance problems
 
+:::note                              
+Starting in 3.29, Flutter on Android and iOS execute Dart code
```

Comment:
We probably _don't_ need a note here, because even though the threads will be changed, the tooling will still communicate that these work is happening on the "UI thread", the devtools traces will be the same, et cetera. Its just that this thread happens to be the same as the platform main thread.